src/components/parts/_dynamic/filters-menu/FilterCheckbox.less

```diff
@@ -0,0 +1,90 @@
+@import (reference) '../../../../global';
+@SuccessStroke: '~@navikt/ds-icons/svg/SuccessStroke.svg';
+
+.filter-checkbox {
+    &__wrapper {
+        background-color: transparent;
+        border-radius: 4px;
+        box-shadow: @navBla inset 0 0 0 1.1px;
+        cursor: pointer;
+        display: flex;
+        justify-items: center;
+        margin: 5px 7px 5px 0;
+        padding: 5px 7px 5px 5px;
```

Comment:
Som regel brukes rem-enheter for margin/padding/height/width/etc, slik at disse skalerer med font-størrelse satt i browser.